parts/k8s/cloud-init/artifacts/rotate-certs.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash -ex
```

Comment:
How is this script delivered to the host VM?

Reply:
scp, moving files out of `cloud-init` dir to avoid confusion